src/components/Header/components/GlobalMenu/index.js

```diff
@@ -43,6 +43,7 @@ export default class GordonGlobalMenu extends Component {
         >
           <MenuItem onClick={this.onClose}><Link to="/help">Help</Link></MenuItem>
           <MenuItem onClick={this.onClose}><Link to="">Logout</Link></MenuItem>
+          <MenuItem onClick={this.onClose}><Link to="/about">About</Link></MenuItem>
```

Comment:
By convention I've seen on other sites, I think Logout should be last on this list, and probably also separated by a divider. You can use the [`Divider`](https://material-ui-next.com/demos/dividers/) component for that.